tests/ui/js/storyTellingServerUITests.js

```diff
@@ -69,98 +69,94 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
-                    task: "sjrk.storyTelling.storyTellingServerUiTester.loadThemedPageSingleTest",
-                    args: ["sjrk.storyTelling.testUtils.callbackVerificationFunction", loadThemedPageTestCases.learningReflections.theme],
+                    task: "sjrk.storyTelling.loadTheme",
                     resolve: "jqUnit.assertDeepEq",
-                    resolveArgs: ["The themed page load resolved as expected", loadThemedPageTestCases.learningReflections, "{arguments}.0"]
+                    resolveArgs: ["The themed page load resolved as expected", "{that}.options.baseTestCase.clientConfig", "{arguments}.0"]
                 },{
-                    funcName: "sjrk.storyTelling.storyTellingServerUiTester.assertCustomCssLoaded",
-                    args: ["learningReflections.css", 1]
+                    funcName: "sjrk.storyTelling.storyTellingServerUiTester.teardownMockServer"
+                }]
+            },{
+                name: "Test themed page loading functions with server config values",
+                expect: 3,
+                sequence: [{
+                    task: "$.get",
+                    args: ["/clientConfig"],
+                    resolve: "fluid.set",
+                    resolveArgs: ["{that}", "clientConfig", "{arguments}.0"]
                 },{
-                    // test the CSS/JS injection function directly
-                    funcName: "sjrk.storyTelling.loadCustomThemeFiles",
-                    args: ["sjrk.storyTelling.testUtils.callbackVerificationFunction", {"theme": "learningReflections"}]
+                    task: "sjrk.storyTelling.loadTheme",
+                    resolve: "jqUnit.assertEquals",
+                    resolveArgs: ["The themed page load resolved as expected", "{that}.clientConfig.theme", "{arguments}.0.theme"]
                 },{
-                    funcName: "sjrk.storyTelling.storyTellingServerUiTester.assertCustomCssLoaded",
-                    args: ["learningReflections.css", 2]
+                    funcName: "sjrk.storyTelling.storyTellingServerUiTester.verifyCustomCssLoaded",
+                    args: ["{that}.clientConfig", 1]
+                },{
+                    task: "sjrk.storyTelling.loadCustomThemeFiles",
+                    args: ["{that}.clientConfig"],
+                    resolve: "sjrk.storyTelling.storyTellingServerUiTester.verifyCustomCssLoaded",
+                    resolveArgs: ["{that}.clientConfig", 2]
                 }]
             }]
         }]
     });
 
-    sjrk.storyTelling.storyTellingServerUiTester.loadThemedPageSingleTest = function (callback, expectedTheme) {
-        var loadPromise = fluid.promise();
-
-        sjrk.storyTelling.storyTellingServerUiTester.setupMockServer("/clientConfig", JSON.stringify({
-            theme: expectedTheme,
-            baseTheme: "base",
-            authoringEnabled: true
-        }));
+    sjrk.storyTelling.storyTellingServerUiTester.loadClientConfigFromServer = function (url) {
```

Comment:
I don't see this function being used anywhere. I think it's usage was removed in some of the refactoring and this can probably be dropped.